WIP-CHANGELOG.md

```diff
@@ -1,91 +1,74 @@
 ### Release notes
```

Comment:
Shouldn't this file be empty?

Reply:
Is this perhaps done by the script mentioned in step 3?
https://github.com/elastisys/compliantkubernetes-apps/tree/main/release#major-and-minor-releases

Reply:
Fixed by running script.